indico/modules/events/sessions/templates/management/_session_list.html

```diff
@@ -43,13 +43,20 @@
                                 <span class="vertical-aligner">{{ sess.code }}</span>
                             </td>
                             <td class="i-table">
-                                <span class="vertical-aligner">
-                                    {%- if sess.is_poster %}
-                                        {%- trans %}Poster{% endtrans -%}
-                                    {% else %}
-                                        {%- trans %}Standard{% endtrans -%}
-                                    {% endif -%}
-                                </span>
+                                <a class="i-button {{ 'session-type-picker' if not event.is_locked else 'disabled' }}"
+                                   data-items="{{ types | tojson | forceescape }}"
+                                   data-href="{{ url_for('.session_rest', sess) }}"
+                                   data-method="PATCH"
+                                   data-selected-item-id="{{ (sess.type.id if sess.type else None) | tojson }}">
```

Comment:
lowercase `none` in jinja

Reply:
theoretically you should use ` | forceescape` here as well, but since it's always a number or null it doesn't make a difference